src/multi_vector_simulator/utils/data_parser.py

```diff
@@ -0,0 +1,182 @@
+import pprint
+import logging
+
+from multi_vector_simulator.utils.constants_json_strings import (
+    PROJECT_DATA,
+    ECONOMIC_DATA,
+    SIMULATION_SETTINGS,
+    CONSTRAINTS,
+    ENERGY_CONSUMPTION,
+    ENERGY_CONVERSION,
+    ENERGY_PRODUCTION,
+    ENERGY_STORAGE,
+    ENERGY_BUSSES,
+    ENERGY_PROVIDERS,
+    UNIT,
+    LABEL,
+    OEMOF_ASSET_TYPE,
+    ENERGY_VECTOR,
+    INFLOW_DIRECTION,
+    CONNECTED_CONSUMPTION_SOURCE,
+    CONNECTED_FEEDIN_SINK,
+    ENERGY_PRICE,
+    FEEDIN_TARIFF,
+    PEAK_DEMAND_PRICING,
+    PEAK_DEMAND_PRICING_PERIOD,
+    RENEWABLE_SHARE_DSO,
+    INPUT_BUS_NAME,
+    OUTPUT_BUS_NAME,
+    OUTFLOW_DIRECTION,
+    DEVELOPMENT_COSTS,
+    DISPATCH_PRICE,
+    DISPATCHABILITY,
+    INSTALLED_CAP,
+    LIFETIME,
+    MAXIMUM_CAP,
+    OPTIMIZE_CAP,
+    SPECIFIC_COSTS,
+    SPECIFIC_COSTS_OM,
+    TIMESERIES,
+    AGE_INSTALLED,
+    RENEWABLE_ASSET_BOOL,
+    EFFICIENCY,
+    INPUT_POWER,
+    OUTPUT_POWER,
+    STORAGE_CAPACITY,
+    PROJECT_ID,
+    PROJECT_NAME,
+    SCENARIO_ID,
+    SCENARIO_NAME,
+    START_DATE,
+    EVALUATED_PERIOD,
+    TIMESTEP,
+)
+
+pp = pprint.PrettyPrinter(indent=4)
+
+MAP_EPA_MVS = {
+    "economic_data": ECONOMIC_DATA,
+    "energyProviders": ENERGY_PROVIDERS,
+    "energy_busses": ENERGY_BUSSES,
+    "energy_consumption": ENERGY_CONSUMPTION,
+    "energy_conversion": ENERGY_CONVERSION,
+    "energy_production": ENERGY_PRODUCTION,
+    "energy_storage": ENERGY_STORAGE,
+    "project_data": PROJECT_DATA,
+    "simulation_settings": SIMULATION_SETTINGS,
+    "energy_vector": ENERGY_VECTOR,
+    "installed_capacity": INSTALLED_CAP,
+    "optimize_capacity": OPTIMIZE_CAP,
+    "maximum_capacity": MAXIMUM_CAP,
+    "input_timeseries": TIMESERIES,
+    "constraints": CONSTRAINTS,
+    "renewable_asset": RENEWABLE_ASSET_BOOL,
+}
+
+MAP_MVS_EPA = {value: key for (key, value) in MAP_EPA_MVS.items()}
+
+# Fields expected for parameters of json returned to EPA
+EPA_PARAM_KEYS = {
+    PROJECT_DATA: [PROJECT_ID, PROJECT_NAME, SCENARIO_ID, SCENARIO_NAME,],
+    SIMULATION_SETTINGS: [START_DATE, EVALUATED_PERIOD, TIMESTEP],
+}
+
+# Fields expected for assets' parameters of json returned to EPA
+EPA_ASSET_KEYS = {
+    ENERGY_PROVIDERS: [
+        "asset_type",
+        LABEL,
+        OEMOF_ASSET_TYPE,
+        "energy_vector",
+        INFLOW_DIRECTION,
+        OUTFLOW_DIRECTION,
+        CONNECTED_CONSUMPTION_SOURCE,
+        CONNECTED_FEEDIN_SINK,
+        DEVELOPMENT_COSTS,
+        DISPATCH_PRICE,
+        ENERGY_PRICE,
+        FEEDIN_TARIFF,
+        "installed_capacity",
+        LIFETIME,
+        "optimize_capacity",
```

Comment:
Arent `energy_vector`, `installed_capacity`, `optimize_capacity` also constant variables? Or are you using EPA names here?

Reply:
These are EPA names, I wanted to specifically not have constant variables for EPA names, so that we see the ones that are different and we are aware of them. It is also helpful in the future to know which constant variable values need to be adapted as they differ than the EPA ones